pglookout/pglookout.py

```diff
@@ -154,6 +154,15 @@ def load_config(self, _signal=None, _frame=None):
         self.replication_lag_failover_timeout = self.config.get("max_failover_replication_time_lag", 120.0)
         self.replication_catchup_timeout = self.config.get("replication_catchup_timeout", 300.0)
         self.missing_master_from_config_timeout = self.config.get("missing_master_from_config_timeout", 15.0)
+
+        if self.replication_lag_warning_boundary >= self.replication_lag_failover_timeout:
+            msg = "Replication lag warning boundary (%s) is not lower than its failover timeout (%s)"
+            msg = msg % (self.replication_lag_warning_boundary, self.replication_lag_failover_timeout)
```

Comment:
I wouldn't use the `%` format operator in any new code. Why are you anyway formatting the string separately instead of just doing `self.log.warning("...", parameters)`?

Reply:
just trying to shorten the line length, Will change to what you suggested